index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <title>Fork Me! FCC: Test Suite Template</title>
+    <title>TRIAL TEST</title>
```

Comment:
Please kindly use lower  case letters for your title subject. This will have a normal and more professional appearance. Its usually considered shouting when you use upper case letters in your title. Please kindly edit that.